opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -192,6 +198,15 @@ impl<R: RuntimeChannel> LogProcessor for BatchLogProcessor<R> {
     }
 
     fn shutdown(&self) -> LogResult<()> {
+        let dropped_logs = self.dropped_logs_count.load(Ordering::Relaxed);
+        if dropped_logs > 0 {
+            otel_warn!(
+                name: "BatchLogProcessor.Shutdown",
+                dropped_logs = dropped_logs,
+                message = "Logs were dropped due to a full buffer."
```

Comment:
I am unsure if it is correct to state that logs were dropped due to Buffer full. It looks like there could be other reasons too?

Reply:
I think the only other reason is the queue is closed; i've updated the messages to reflect that